core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2514,9 +2514,8 @@ private SqlDialect nonOrdinalDialect() {
    */
   @Test void testCastArrayCharset() {
     final String query = "select cast(array['a', 'b', 'c'] as varchar array)";
-    final String expected = "SELECT CAST(ARRAY ('a', 'b', 'c') AS VARCHAR ARRAY)";
     sql(query)
-        .withHive().ok(expected);
+        .withHive().throws_("Hive dialect does not support cast to ARRAY");
```

Comment:
@JiajunBernoulli you wrote this test originally, is it indeed incorrect?

Reply:
@mihaibudiu Yes, hive does not support it.

This is my mistake.

Postgre support it, can we add one line `withPostgresql()`?

Reply:
OK，had a test for postgresql and test it work well, thanks for your reply. @JiajunBernoulli